tokio/src/runtime/handle.rs

```diff
@@ -213,7 +213,7 @@ impl Handle {
         #[cfg(not(all(tokio_unstable, feature = "tracing")))]
         let _ = name;
 
-        let (task, handle) = task::joinable(fut);
+        let (task, handle) = task::joinable(fut, NoopSchedule);
```

Comment:
It looks like this is the only place `task::joinable(...)` is used still. Is that because there is no set of owned tasks here?

Reply:
Yes, it's only used for blocking tasks and tests.